make/ingress/run

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+
+if [ -z "${INGRESS_CONTROLLER}" ]; then
```

Comment:
Wouldn't this look better as:

```bash
case "${INGRESS_CONTROLLER}" in
  "")
    echo "No Ingress Controller specified; set INGRESS_CONTROLLER"
    ;;
  nginx)
    "${GIT_ROOT}/make/ingress/nginx/run"
    ;;
  *)
    echo "Ingress controller '${INGRESS_CONTROLLER}' not recognized" >&2
    exit 1
esac
```

Alternatively:
```bash
if [ -z "${INGRESS_CONTROLLER}" ]; then
  echo "No Ingress Controller specified; set INGRESS_CONTROLLER"
  exit
fi

if [ -x "${GIT_ROOT}/make/${INGRESS_CONTROLLER}/run" ] ; then
  exec "${GIT_ROOT}/make/${INGRESS_CONTROLLER}/run" "$@"
fi

echo "Ingress controller '${INGRESS_CONTROLLER}' not recognized" >&2
exit 1
```

(Actually, do we even want the empty string message, if `make run` will run this by default with no controllers?)